offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -123,8 +123,8 @@ class QuotationDetails {
         addSubTotalPrices(productGroup, subTotal)
     }
 
-    private static String createDiscountDescription() {
-        return "Quantity discount."
+    private String createDiscountDescription(double quantity, String unit, double discountPercentage) {
+        return "Discount on ${quantity} ${unit} based on item no ${itemNumber}. ${discountPercentage}% discount applied"
```

Comment:
I think the unit should be different here. Lower case and with a tailing s.
Otherwise it says `Discount for 41.0 Sample`

Reply:
```suggestion
        String unitName = unit.toString().toLowerCase()
        unitName = quantity != 1 : unitName + "s" : unitName
        return "Discount on ${quantity} ${unitName} based on item no ${itemNumber}. ${discountPercentage}% discount applied"
```